lib/begin.go

```diff
@@ -181,6 +193,15 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if urlerr, ok := err.(*url.Error); ok {
+			if operr, ok := urlerr.Err.(*net.OpError); ok {
+				if dnserr, ok := operr.Err.(*net.DNSError); ok {
+					if dnserr.Err == "no such host" {
```

Comment:
I'm modifying begin such that when a local file is specified as a starting point, it must begin with `/` or `.`. This is so that I can return a nicer error message when the local file isn't found. This introduces an issue of detecting when the user wants to use a local file but doesn't start the argument with the correct character. The way acbuild handles this is attempting to do meta discovery on the string and then failing when the DNS lookup fails. This is attempting to determine when that error happens. @jonboulle didn't like me doing `strings.Contains(err.Error(), "no such host")`, so this was the result.